src/Listing/index.tsx

```diff
@@ -127,11 +126,10 @@ const Listing = ({
   onSort = (): void => undefined,
   labelDisplayedRows = ({ from, to, count }): string =>
     `${from}-${to} of ${count}`,
+  getCompositeId = ({ id }) => [id, id],
```

Comment:
```suggestion
  getCompositeId = ({ id }) => [id],
```
Maybe the default value should be a simple id